js/foundation.offcanvas.js

```diff
@@ -336,22 +336,7 @@ class OffCanvas extends Plugin {
    * @private
    */
   _recordScrollable(event) {
-    let elem = this; // called from event handler context with this as elem
-
-     // If the element is scrollable (content overflows), then...
-    if (elem.scrollHeight !== elem.clientHeight) {
-      // If we're at the top, scroll down one pixel to allow scrolling up
-      if (elem.scrollTop === 0) {
-        elem.scrollTop = 1;
-      }
-      // If we're at the bottom, scroll up one pixel to allow scrolling down
-      if (elem.scrollTop === elem.scrollHeight - elem.clientHeight) {
-        elem.scrollTop = elem.scrollHeight - elem.clientHeight - 1;
-      }
-    }
-    elem.allowUp = elem.scrollTop > 0;
-    elem.allowDown = elem.scrollTop < (elem.scrollHeight - elem.clientHeight);
-    elem.lastY = event.originalEvent.pageY;
+    this.lastY = event.touches[0].pageY;
```

Comment:
Does that even work in non-touch context?

Reply:
The handlers are called only in touch context
https://github.com/foundation/foundation-sites/blob/aa82df837f324205a8233bae3fa8e1ae828eba42/js/foundation.offcanvas.js#L430-L433